actions/st2_perform_migrations.sh

```diff
@@ -153,6 +153,23 @@ migrate_to_1.5() {
   run_migration_scripts $scripts
 }
 
+migrate_to_2.2() {
+  sudo service mistral-api stop
+  sudo service mistral stop
+  sudo -u postgres psql
+  \connect mistral
+  DROP TABLE event_triggers_v2;
+  DROP TABLE workflow_executions_v2 CASCADE;
+  DROP TABLE task_executions_v2;
+  DROP TABLE action_executions_v2;
+  DROP TABLE named_locks;
+  \q
+  /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head
+  /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate
+  sudo service mistral start
+  sudo service mistral-api start
+}
+
```

Comment:
Looks like it tries to find a function according to the current upgraded version:

https://github.com/StackStorm/st2cd/pull/251/files#diff-724355fdedcf4c8f85a34f74855378bdL124

That's why I named mine `migrate_to_2.2` - currently, it's not finding an upgrade function so it's exiting with the message `"--> No migrations to run for version $migration_ver"`